src/provisioner/provisioner.py

```diff
@@ -127,7 +127,7 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
                     Conf.set(CortxProvisioner._solution_index, key, val.decode('utf-8'))
             CortxProvisioner.apply_cortx_config(cortx_conf, CortxProvisioner.cortx_release)
             # Adding array count key in conf
-            cortx_conf.add_num_keys()
+            CortxProvisioner._add_num_keys(cortx_conf)
```

Comment:
For two lines its worth to put them here for readability and fact that explicit save is called. 
```
        cortx_conf.add_num_keys()
        Conf.save(cortx_conf._conf_idx)
```

Reply:
@ujjwalpl  correct, but for unit test convenience I have separated the concern of adding num_xxx keys to a different function. Please confirm if you still want this change.

Reply:
done